volatility3/framework/symbols/linux/__init__.py

```diff
@@ -830,3 +834,97 @@ def get_cached_pages(self) -> interfaces.objects.ObjectInterface:
             page = self.vmlinux.object("page", offset=page_addr, absolute=True)
             if page:
                 yield page
+
+
+@dataclasses.dataclass
+class TimespecVol3(object):
+    """Internal helper class to handle all required timespec operations, convertions and
+    adjustments.
+
+    NOTE: This is intended for exclusive use with get_boottime() and its related functions.
```

Comment:
Why... then... isn't it in a boottime specific class?  None of these methods seem very specific to `boottime`, they seem specific to timespec64, which is a linux specific extension.

Reply:
I'd like to hear your thoughts on creating a timespec64 dummy object. For now, I prefer to keep this 'private', for the same reasons you've mentioned in your other comments. I wasn't particularly keen on creating a class that seems 'outside' the framework, but I couldn't find another way to resolve it